bodhi/server/consumers/masher.py

```diff
@@ -902,6 +903,36 @@ def compose_atomic_trees(self):
         mashed_repos = dict([('-'.join(os.path.basename(repo).split('-')[:-1]), repo)
                              for repo in self.state['completed_repos']])
         for tag, mash_path in mashed_repos.items():
+            #  TODO: Find out the actual values for the tag
+            #  also for the code below, should we use the mash_path instead
+            #  of current_workdir?
+            #  * Pungi config repos hsa $arch, does it matter in bodhi?
```

Comment:
@bowlofeggs that's sort of what I meant: If Bodhi creates separate repos for each arch, then the `$arch` placeholder in the URL is needed. If Bodhi creates just one big repo with all architectures, the URL for Pungi should not contain it.